src/repository/msgraph/event.repository.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
```

Comment:
call-urile externe le faci in service class, nu repository :) .
Repository se refera doar la interogarea Db-ului si manipularea entitatiilor.

Reply:
am mutat call-urile externe din repository in service files